meson.build

```diff
@@ -190,6 +190,7 @@ if get_option('enable-dell')
 endif
 
 if get_option('enable-thunderbolt')
+  fwup = dependency('fwup', version : '>= 5')
```

Comment:
I think the thunderbolt plugin doesn't actually need that. I think it is safe to remove the dependency from the plugin's `meson.build` instead.